ui-tests/tests/0_basic_ray.test.ts

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
```

Comment:
You give the Jupyter dev team too much credit. I think we can take credit for this one. Copying the copyright from one of our other files and adjusting the year should be fine